lib/irc/BridgedClient.js

```diff
@@ -342,8 +346,35 @@ BridgedClient.prototype.whois = function(nick) {
  * Get the operators of a channel
  * @param {string} channel : The channel to call /names on
  * @param {string} key : Optional. The key to use to join the channel.
+ * @param {integer} cacheDurationMs : Optional. The duration of time to keep a
+ * list of operator nicks cached. If > 0, the operator nicks will be returned
+ * whilst the cache is still valid and it will become invalid after cacheDurationMs
+ * milliseconds. Cache will not be used if left undefined.
  */
-BridgedClient.prototype.getOperators = function(channel, key) {
+BridgedClient.prototype.getOperators = function(channel, key, cacheDurationMs) {
```

Comment:
Rather than these two positional arguments (and the complications of detecting which by type) it might be nicer to pass in `opts` in the usual named-argument style. Also makes callsites neater with a `{timeout: 10000}` rather than just a trailing number.